src/main/java/de/dataelementhub/model/handler/element/DataElementGroupHandler.java

```diff
@@ -102,4 +102,28 @@ public static Identification update(CloseableDSLContext ctx, int userId,
     return dataElementGroup.getIdentification();
   }
 
+  /**
+   * Update dataElementGroup members.
+   * If changes acquired return the new identification otherwise return the old identification.
```

Comment:
Not sure if "acquired" is the correct word. "Occurred" would be better imho. Also, in this case, use the @returns annotation